src/main.js

```diff
@@ -241,6 +242,16 @@ function saveHelper() {
   savePoster();
 }
 
+function deleteOnClick(event) {
+ var posterId = parseInt(event.target.id)
+ for (var i = 0; i < savedPosters.length; i++) {
+   if (savedPosters[i].id === posterId) {
+     savedPosters.splice(i, 1)
+     saveHelper();
+   }
+ }
+}
+
```

Comment:
This is great. Only suggestion would be to consider what information the function name provides to the reader not only where the function is defined, but where it may be passed or invoked. Would be useful for other developers to quickly know what this function deletes.